data/data/powervs/main.tf

```diff
@@ -66,10 +66,15 @@ data "ibm_is_subnet" "vpc_subnet" {
   name     = var.powervs_vpc_subnet_name
 }
 
-data "ibm_pi_image" "boot_image" {
-  provider             = ibm.powervs
-  pi_image_name        = var.powervs_image_name
-  pi_cloud_instance_id = var.powervs_cloud_instance_id
+resource "ibm_pi_image" "boot_image" {
+  provider                  = ibm.powervs
+  pi_image_name             = "${var.cluster_id}-boot-image"
+  pi_cloud_instance_id      = var.powervs_cloud_instance_id
+  pi_image_bucket_name      = "rhcos-powervs-images-${var.powervs_vpc_region}"
+  pi_image_bucket_access    = var.powervs_image_bucket_access
```

Comment:
we an hard-code this to `public` since that (public) is a requirement for the import API

Reply:
done!